roofit/roofitcore/inc/RooAbsArg.h

```diff
@@ -617,8 +619,8 @@ class RooAbsArg : public TNamed, public RooPrintable {
   // Proxy management
   friend class RooAddModel ;
   friend class RooArgProxy ;
-  friend class RooSetProxy ;
-  friend class RooListProxy ;
+  template<class RooCollection_t>
+  friend class RooCollectionProxy;
```

Comment:
Try without these friend declarations if possible. That's a code smell. Sometimes it was possible to remove them, because old code had already been modernised.